packages/fuels-core/src/types/bits_256.rs

```diff
@@ -68,6 +68,7 @@ impl EvmAddress {
 
 impl From<Bits256> for EvmAddress {
     fn from(b256: Bits256) -> Self {
+        // An evm address is only 20 bytes
```

Comment:
Is there a spec refering to the fact that we decided that the bytes to clear where the leftmost ones?